application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -36,300 +36,282 @@
+              #displayInput([] true $xcontext.user)
+            &lt;/tr&gt;
+          #end
+          &lt;/tbody&gt;
+
+          ## Votes count.
+          #set ($totalVotesCount = 0)
+          #set ($voteCount = $services.xpoll.getVoteResults($doc.documentReference))
+          #foreach ($key in $voteCount.keySet())
+            #set ($totalVotesCount = $totalVotesCount + $voteCount.get($key))
+          #end
+          &lt;tfoot&gt;
+            &lt;tr&gt;
+              &lt;td&gt;
+                $services.localization.render('contrib.xpoll.numberVotes')
+                &lt;span class='count'&gt;($totalVotesCount)&lt;/span&gt;
+              &lt;/td&gt;
+              #foreach ($proposal in $proposals) &lt;td&gt; $!voteCount.get($proposal) &lt;/td&gt; #end
+            &lt;/tr&gt;
+          &lt;/tfoot&gt;
+        &lt;/table&gt;
+        #if ($xcontext.user != 'XWiki.XWikiGuest')
+          &lt;div class='save'&gt;
+            &lt;input type="submit" value="$services.localization.render("contrib.xpoll.vote.user.submit")"
+              class="button" /&gt;
+          &lt;/div&gt;
+        #end
+      &lt;/form&gt;
+    {{/html}}
+  #elseif ($status == 'finished')
+
+    {{html wiki='true'}}
+      $services.localization.render('contrib.xpoll.finish.message')
+      &lt;table class='xpoll medium-avatars table table-bordered'&gt;
+        ## Table header row.
+        &lt;tr&gt;
+          &lt;th&gt;
+            $services.localization.render('contrib.xpoll.user')
+            &lt;span class='count'&gt;($usersCount)&lt;/span&gt;
+          &lt;/th&gt;
+          #foreach ($proposal in $proposals)&lt;th&gt; $proposal &lt;/th&gt; #end
+        &lt;/tr&gt;
+
+        ## Users and their votes.
+        #foreach ($voteObj in $doc.getObjects('XPoll.XPollVoteClass'))
+          #set ($user = $voteObj.user)
+          #set ($votes = $voteObj.votes)
+          #set ($isCurrentUser = $user == $xcontext.user)
+          &lt;tr #if($isCurrentUser)class="active"#end&gt;
+            &lt;td&gt;
+              #displayUser($user)
+            &lt;/td&gt;
+            ## The 3rd parameter is false in order to disable the input when the poll is finished.
+            #displayInput($votes false $user)
+          &lt;/tr&gt;
+        #end
+
+        ## Votes count.
+          #set ($totalVotesCount = 0)
+          #set ($voteCount = $services.xpoll.getVoteResults($doc.documentReference))
+          #foreach ($key in $voteCount.keySet())
+            #set ($totalVotesCount = $totalVotesCount + $voteCount.get($key))
+          #end
+        &lt;tr&gt;
+          &lt;td&gt;
+            $services.localization.render('contrib.xpoll.numberVotes')
+            &lt;span class='count'&gt;($totalVotesCount)&lt;/span&gt;
+          &lt;/td&gt;
+          #foreach ($proposal in $proposals)
+            &lt;td&gt; $!voteCount.get($proposal) &lt;/td&gt;
+          #end
+        &lt;/tr&gt;
+      &lt;/table&gt;
+    {{/html}}
+  #end
+#end

```

Comment:
Did you run ``mvn xar:format``? There are too many changes reported (which makes it hard to review) and I don't see that many differences so I guess it's because of the line endings?